packages/ses/test/test-scope.js

```diff
@@ -370,19 +373,27 @@ test('scope behavior - Symbol.unscopables fidelity test', t => {
   );
   // after property is created on globalObject, assignment is evaluated to
   // test if it is affected by the Symbol.unscopables configuration
+  // Known compromise in fidelity of the emulated script environment:
```

Comment:
How is it a fidelity compromise in this case? This would only be possible in sloppy mode, no?

Reply:
seems like this comment was added here by accident